bot/cogs/snakes.py

```diff
@@ -15,31 +38,67 @@ class Snakes:
     def __init__(self, bot: AutoShardedBot):
         self.bot = bot
 
+    @staticmethod
+    def snake_url(name) -> str:
+        """Get the URL of a snake"""
+
+        def encode_url(text):
+            """Encode a string to URL-friendly format"""
+            return BASEURL.format(text.replace(' ', '%20').replace("'", '%27'))
+
+        # Check if the snake name is known
+        if name.upper() in list(map(lambda n: n.upper(), SNAKES)):
+            return encode_url(name)
+
+        # Get a list of similar names if a match wasn't found
+        matches = difflib.get_close_matches(name, SNAKES, n=1)
+        return encode_url(matches[0] if matches != [] else random.choice(SNAKES))
+
+    @staticmethod
+    async def fetch(session, url):
+        """Fetch the contents of a URL as a json"""
+        async with async_timeout.timeout(10):
+            async with session.get(url) as response:
+                return await response.json()
+
     async def get_snek(self, name: str = None) -> Dict[str, Any]:
-        """
-        Go online and fetch information about a snake
+        """Get a snake with a given name, or otherwise randomly"""
```

Comment:
Poorly written docstring. "otherwise randomly".  

`"If a name is provided, this gets a specific snake. Otherwise, it gets a random snake."`